### ISSUE TITLE
Missing clarity about JWS requirement

### ISSUE THREAD

**Issue by qligier** [IHE/ITI.IUA#90]
**Section Number** 3.71.4.2.2.1

**Issue**
It is unclear whether JWT shall be or may be signed.
The profile says _JWT token shall be signed as specified in JSON Web Signature [RFC7515]_, which would require the use of JWS, but the next sentence starts with _If signed_. The first sentence could then be understood as "If signed, one must follow RFC7515" (i.e. a restriction on the signature method, and not an unconditional requirement).

**Proposed Change**
N/A

**Priority**:
N/A

**Comment by JohnMoehrke** [comment]
Martin, can you review and comment?

**Comment by msmock** [comment]
I agree. We should remove the "If signed" phrase from the sentence. This is not a functional change, so I guess we don't need a ballot.

**Comment by JohnMoehrke** [comment]
excellent, please submit a pull-request. We can review during a meeting and approve if the committee also sees it as a technical correction.

**Comment by JohnMoehrke** [comment]
A PR to resolve this is needed.

**Comment by qligier** [comment]
I think removing the `If signed` would be a functional change because of the next paragraph that specifies:

> Any actor may support the JWE (unsigned but encrypted) alternative of the JWT token.

This makes it clear the signature is not unconditionally required.

I think the `If signed` should be moved at the beginning of the paragraph:

> **If the JWT token is signed**, it must be done in accordance with JSON Web Signature [RFC7515]; in particular, the JWS Compact Serialization (base64 encoded, with single signature or MACed) shall be used as described in [RFC7515, Section 7.1].
> 
> Any actor may support the JWE (unsigned but encrypted) alternative of the JWT token.

**Comment by JohnMoehrke** [comment]
do we want to say that it is not required by IUA, but may be required by policies? That explains that we don't have an IHE requirement, but that it still might be a requirement for environments that decided it is necessary.

**Comment by JohnMoehrke** [comment]
The PR seems to enable non-signed. I thought the recommendation was to make clear it must be signed?

Also, why do we have the sentence after this? "Any actor may support the JWE (unsigned but encrypted) alternative of the JWT token."
